docs/UserGuide.md

```diff
@@ -372,11 +372,12 @@ Lets you add an internship entry to be tracked by CareerSync!
 
 **Format:** `add /com COMPANY_NAME /desc DESCRIPTION /status STATUS /poc CONTACT_NAME /email CONTACT_EMAIL /phone CONTACT_NUMBER [/loc LOCATION_ENUM] [/role ROLE] ​` <br>
 
-* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` and `ROLE` allow the use of any text, number and/or spaces
+* The fields `COMPANY_NAME`, `DESCRIPTION`, `CONTACT_NAME` allow the use of any text, number and/or spaces
 * The field `STATUS` accepts only the following inputs: `to_apply`, `ongoing`, `rejected`, `accepted` (case-insensitive)
 * The field `CONTACT_EMAIL` accepts only the format of `<email>@<domain>.com`
 * The field `CONTACT_NUMBER` must be unsigned, and it must be at least 7 digits long
-* The field `LOCATION_ENUM` accepts only the following inputs: `remote`, `local`, `overseas` (case-insensitive)
+* The field `LOCATION_ENUM` accepts only the following inputs: `unknown`, `remote`, `local`, `overseas` (case-insensitive)
+* The field `ROLE` allow the use of any text, number and/or spaces
```

Comment:
Maybe an easier description would be any ASCII character EXCEPT '!' and '?', now it doesnt include the fact that you can have symbols.